src/tests/suite/SConscript

```diff
@@ -105,13 +105,13 @@ def scons():
                     daos_aggregate_ec.c daos_degrade_ec.c
                     daos_extend_simple.c daos_obj_ec.c""")
 
-    newenv.AppendUnique(LIBPATH=["$BUILD_DIR/src/client/dfs"])
+    newenv.AppendUnique(LIBPATH=[Dir('../../client/dfs')])
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string